scripts/ci/k3d.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+set -e
+set -u
+set -o pipefail
+
+# Install k3d if not found
+if ! command -v k3d &> /dev/null
+then
+    curl -s https://raw.githubusercontent.com/rancher/k3d/main/install.sh | bash
+fi
```

Comment:
I know the script is already simple, but it might be good idea to split k3d installation from cluster creation to separate make targets

Reply:
done.